modules/persistent-storage-local-metrics.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-storage-metrics_{context}"]
+= Local Storage Operator Metrics
+
+{product-title} provides the following metrics for the Local Storage Operator:
+
+* lso_discovery_disk_count: total number of discovered devices on each node
```

Comment:
```suggestion
* `lso_discovery_disk_count`: total number of discovered devices on each node
```

This is the snippet from our doc guidelines. This comment applies to all of the bullets in the list: 
> Use backticks for all non-GUI "system items", including: Inline commands, operations, literal values, variables, parameters, settings, flags, environment variables, user input. System term/item, user names, unique or example names for individual API objects/resources (e.g., a pod named mypod), daemon, service, or software package. RPM packages. File names or directory paths